addons/web_editor/models/ir_ui_view.py

```diff
@@ -134,19 +130,25 @@ def _copy_field_terms_translations(self, record_from, name_field_from, record_to
         if not field_to.store:
             raise ValueError("Field %r is not stored" % field_to)
 
+        # This will also implicitly check for `read` access rights
+        if not record_to[name_field_to] or not any(recordset_from.mapped(name_field_from)):
+            return
```

Comment:
probably not important but any reason to move this below, and not keep it "grouped" with other access check?

Reply:
yes, The rationale was to move the raises above, to prevent reading stuff from DB (if the fields weren't in cache yet, a read query would have been passed) if it was not necessary. Given that the record_to[field_to] has a value, we can prevent reading the record_from, which, in the case of views, can be rather big.
Now that you mention it, we can do the checks on the registry (field.translate) even before any check on values from the db (read access and rules). What do you think ?